internal/gatewayapi/securitypolicy.go

```diff
@@ -502,6 +502,7 @@ func (t *Translator) buildOIDC(
 		RedirectURL:  redirectURL,
 		RedirectPath: redirectPath,
 		LogoutPath:   logoutPath,
+		CookieSuffix: fmt.Sprintf("%s-%s", policy.Namespace, policy.Name),
```

Comment:
feels like config leak

Reply:
what about something like a hash on the clientID or policyNsName (stable yet irreversible) ?